Assets/MixedRealityToolkit-Examples/Prototyping/Shaders/UnlitTransparentTriplanar.shader

```diff
@@ -52,9 +54,10 @@ Shader "MixedRealityToolkit/Examples/UnlitTransparentTriplanar"
 			fixed3 _LightDir;
 			float1 _LightIntensity;
 			float1 _CameraIntensity;
-			UNITY_INSTANCING_CBUFFER_START(Props)
+			UNITY_INSTANCING_BUFFER_START(Props)
```

Comment:
Is this compatible with previous versions of unity? 
Eg 5.6, 2017.1?

Reply:
I thought the plan was to only support from 2017.2 from now.  But as this is targeting Dev, should it matter?